src/types/Speaker.ts

```diff
@@ -0,0 +1,11 @@
+//Definition of the speaker type
```

Comment:
Since this interface is only used in`SingleSpeakerScreen.tsx`, you can instead define it inside that file.

Reply:
ok